Front_end/lib/pages/product_detail.dart

```diff
@@ -1,103 +1,161 @@
 import 'package:flutter/material.dart';
-import 'package:front_end/models/photo.dart';
+import 'package:front_end/DTOs/Product.dart';
+import 'package:front_end/DTOs/Pedido.dart';
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+import 'package:front_end/pages/home.dart';
+
+// obj: inserir uma unidade do produto no pedido cujo id seja igual a idPedido, e receber ora um erro a ser tratado, ora um pedido.
+// TODO: ao terminar de implementar as funcionalidades de usuário, alterar parâmetros do método para que ele receba o id do pedido do usuário.
 
 class ProductDetail extends StatefulWidget {
-  ProductDetail(this.photo, {Key? key}) : super(key: key);
+  ProductDetail(this.product, {Key? key}) : super(key: key);
```

Comment:
Sugestão:
`ProductDetail(this.product, {Key? key, required this.carrinhoId}) : super(key: key);`